### ISSUE TITLE
no wifi after flashing to latest version

### ISSUE THREAD

**Issue by aaronmcnees** [spacehuhn/esp8266_deauther#1070]
Hi, I am brand new to this but latest software seems to have made both my Amica and D1 not show up in wifi after flashing no matter what I try to flash afterwards, any and all advice is most welcome. I am using Node MCU Flasher via USB Windows 7. 
Thank you to everyone for making this fascinating project.

**Comment by aaronmcnees** [comment]
Update- switched USB port and it started working again. I don't know why but I'll take it as a win. This was on the D1, now to try and fix the Amica.

**Comment by aaronmcnees** [comment]
Addendum: I got the Amica running again as well, I tried everything several times with no luck but I finally got the wifi connectivity back by flashing the older version deauther program on a different USB port (same COM tho) at the lowest settings (baud, etc) possible except for QIO.
And now its back.
For anyone else with wifi not showing up after flash, this worked for me. I'm still very much an infant in this, but I am learning.
A couple of years ago, I picked up a pair of paperclips and started learning how to pick locks....
Fast forward to now, I make and sell lockpicks full time. And they are some of the very best in the field.
Again, thank you Stefan and Travis for making this project. Rock on.
